src/scenes/levelend-scene.ts

```diff
@@ -1,23 +1,22 @@
-import { loadImages } from "../common/";
-import { CLICK } from "../common/event-names";
-import { AudioPlayer } from "../components/audio-player";
-import CloseButton from "../components/buttons/close-button";
-import NextButton from "../components/buttons/next-button";
-import RetryButton from "../components/buttons/retry-button";
-import { Monster } from "../components/monster";
-import {
-  DEFAULT_BG_GROUP_IMGS,
-  AUTUMN_BG_GROUP_IMGS,
-  WINTER_BG_GROUP_IMGS
-} from '../constants';
+import { loadImages, CLICK } from "@common";
+import { AudioPlayer } from "@components";
+import { CloseButton, NextButton, RetryButton } from "@buttons";
+import { Monster } from "@components";
```

Comment:
Line 2 and 4 are both from components

Reply:
Thanks @bernhardccodev I applied a fix here.